content/reference-rippled.md

```diff
@@ -951,7 +1130,7 @@ Each object in the `transactions` array, if present, may contain any or all of t
 ## account_lines ##
 [[Source]<br>](https://github.com/ripple/rippled/blob/master/src/ripple/rpc/handlers/AccountLines.cpp "Source")
 
-The `account_lines` method returns information about the account's lines of trust, including balances in all non-XRP currencies and assets. All information retrieved is relative to a particular version of the ledger.
+The `account_lines` method returns information about an account's trust lines, including balances in all non-XRP currencies and assets. All information retrieved is relative to a particular version of the ledger.
```

Comment:
Hmmm, I see now that 25 times we do not, but four times we do hyphenate trust lines. Not sure if you want to get those instances in this PR, or pick them up as typos somewhere else.